packages/lang/lua52/config/lua52.pc

```diff
@@ -0,0 +1,20 @@
+V=5.2
```

Comment:
given that there are versions in this file - would prefer to see the hardcoding version # managed via sed / as part of the makefile ... - otherwise if will be missed.